docs/commands/graph.query.md

```diff
@@ -968,39 +970,39 @@ This section contains information on all supported functions from the Cypher que
-| reverse(_str_)                      | Returns a string in which the order of all characters in _str_ are reversed <br> Returns null when _str_ evaluates to null                                     |
+| left(_str_, _len_)                  | Returns a string containing the _len_ leftmost characters of _str_ <br> Returns null when _str_ evaluates to null, otherwise error if _len_ evaluates to null |
+| lTrim(_str_)                        | Returns _str_ with leading whitespace removed <br> Returns null when _str_ evaluates to null                                                                     |
+| replace(_str_, _search_, _replace_) | Returns _str_ with all occurrences of _search_ replaced with _replace_ <br> Returns null when any argument evaluates to null                                     |
+| reverse(_str_)                      | Returns a string in which the order of all characters in _str_ are reversed <br> Returns null when _str_ evaluates to null                                       |
 | right(_str_, _len_)                 | Returns a string containing the _len_ rightmost characters of _str_ <br> Returns null when _str_ evaluates to null, otherwise error if _len_ evaluates to null |
-| rTrim(_str_)                        | Returns _str_ with trailing whitespace removed <br> Returns null when _str_ evaluates to null                                                                  |
-| split(_str_, _delimiter_)           | Returns a list of strings from splitting _str_ by _delimiter_ <br> Returns null when any argument evaluates to null                                            |
+| rTrim(_str_)                        | Returns _str_ with trailing whitespace removed <br> Returns null when _str_ evaluates to null                                                                   |
+| split(_str_, _delimiter_)           | Returns a list of strings from splitting _str_ by _delimiter_ <br> Returns null when any argument evaluates to null                                             |
+| string.join(_strList_[, _delimiter_ = '']) *         | (Since RedisGraph v2.12) <br> Returns a concatenation of a list of strings using a given delimiter <br> Returns null when _strList_ evaluates to null <br> Returns null when _delimiter_, if specified, evaluates to null <br> Error when _strList_ does not evaluate to a list or to null <br> Error when an element of _strList_ does not evaluate to a string <br> Error when _delimiter_, if specified, does not evaluate to a string or to null |
+| string.matchRegEx(_str_, _regex_) *                  | (Since RedisGraph v2.12) <br> Given a string and a regular expression, returns a list of all matches and matching regions <br> Returns an empty list when _str_ evaluates to null <br> Returns an empty list when _regex_ evaluates to null <br> Error when _str_ does not evaluate to a string or to null <br> Error when _regex_ does not evaluate to a valid regex string or to null |
```

Comment:
Missing an example,
Would be nice to include a simple example illustrating the returned value structure

Reply:
I plan to add examples for all functions, but not right now.